.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        django_version: ["2.2"]
+        python_version: ["3.6", "3.7", "3.8"]
```

Comment:
Because Django doesn't maintain the same Python version compatibility between all versions, this isn't guaranteed to always work as part of the matrix. However it'll certainly work until we add the next LTS version, and if we're restricting ourselves to a common set of more recent Python versions supported by all LTS versions.

I think we should cross that bridge when we come to it and rely on something like `exclude` for edge cases, instead of getting really verbose with specifying Python versions per version in `include`.

If we wanted something closer to how Travis' matrix syntax works, we could be verbose and state everything as an object as [shown here](https://github.com/actions/runner/issues/343). However that feature is sparsely documented and I'd rather rely on the well-documented `property: [list]` syntax.

Reply:
> If we wanted something closer to how Travis' matrix syntax works, we could be verbose and state everything as an object as [shown here](https://github.com/actions/runner/issues/343). However that feature is sparsely documented and I'd rather rely on the well-documented `property: [list]` syntax.

As it is now, it's hard to understand what the matrix strategy is. Does it just test every possible permutation of combinations of each array?

Reply:
Yep! So it'd build Django `2.2` for Python `3.6`, `3.7`, and `3.8`, and then build each of those for `alpine` and `slim`, for a total of 6 builds.

Additionally, the `include` below will match on anything with `variant == alpine` and set the `PG_MAJOR` and `PG_VERSION` appropriately for each of the matching permutations. Same deal with `slim`.